gpMgmt/bin/gpexpand

```diff
@@ -773,8 +773,19 @@ class SegmentTemplate:
 
     def _distribute_tarfile(self):
         """Distributes template tar file to hosts"""
+        localHostname = getLocalHostname()
+        # NOTE: Currently, GP's scp transfers relative file paths into the home
+        # directory by default.
+        pathsAreSame = (os.path.realpath(self.segTarFile) == os.path.join('/home/%s/' % getUserName(), self.schema_tar_file))
```

Comment:
```suggestion
        pathsAreSame = os.path.realpath(self.segTarFile) == os.path.join('home', getUserName(), self.schema_tar_file)
```

Reply:
why not something like `os.environ.get('HOME', '.')`?

Reply:
This is hidden behind function usually. Then we would need to add a new one called `getHome()` into `gppylib/commands/unix.py`.

Reply:
> we would need to add a new one called `getHome()` into `gppylib/commands/unix.py`.

why not